aspnetcore/security/key-vault-configuration.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Azure Key Vault Configuration Provider | Microsoft Docs
+author: guardrex
+description: How to use the Azure Key Vault Configuration Provider to configure an application using name-value pairs loaded at runtime.
+keywords: ASP.NET Core, configuration, Azure Key Vault
+ms.author: riande
+manager: wpickett
+ms.date: 1/16/2017
+ms.topic: article
+ms.assetid: 0292bdae-b3ed-4637-bd67-19b9bb8b65cb
+ms.prod: aspnet-core
+uid: security/key-vault-configuration
+---
+# Azure Key Vault Configuration Provider
+
+By [Luke Latham](https://github.com/GuardRex) and [Andrew Stanton-Nurse](https://github.com/anurse)
+
+[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/security/key-vault-configuration/sample)
+
+This document explains how to use the Azure Key Vault Configuration Provider to load application configuration values from Azure Key Vault. [Microsoft Azure Key Vault](https://azure.microsoft.com/en-us/services/key-vault/) is a cloud-based service that helps you safeguard cryptographic keys and secrets used by applications and services.
+
+## When to use the provider
+Use the provider when your application has access to Azure services and requires encrypted storage of its configuration data. Common scenarios include:
+* Controlling access to sensitive configuration data
+* Monitoring and logging of configuration data use
+* Reducing latency, providing automatic scaling, and providing redundancy of configuration data
```

Comment:
Let's simplify this to ensure we aren't just duplicating content. I would definitely call out the FIPS compliant part, as well as the centralized access control and tracking of secrets, since those are fundamental reasons to use Key Vault. But perhaps something shorter that calls out those two key benefits, then just links to the Azure Key Vault docs for more scenarios would be more appropriate, especially if there are some we don't quite understand the justification for ;)